notebooks/EDA.ipynb

```diff
@@ -0,0 +1,4712 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    df["school_age_map"] = df["school_age"].map(lambda x: 150 if x &gt;= 150 else x)
</pre><p>Takie przypisanie wprost może być zgubne bo nie wiadomo czy szkoła ma 150 czy 300 lat.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataWorkshop-Foundation/way-to-school/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>